include/avif/avif.h

```diff
@@ -772,6 +795,16 @@ typedef struct avifDecoder
     // Set this via avifDecoderSetSource().
     avifDecoderSource requestedSource;
 
+    // If this is true and a progressive AVIF is decoded, avifDecoder will behave as if the AVIF
+    // is an image sequence, in that it will set imageCount to the available number of progressive
```

Comment:
Delete one of the two "available"s in "the available number of progressive frames available"